build-config/conf/busybox.config

```diff
@@ -777,6 +777,9 @@ CONFIG_WATCHDOG=y
 CONFIG_SYS_EEPROM_I2C_DEVICE=""
 CONFIG_SYS_EEPROM_I2C_ADDR=
 CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=0
+CONFIG_SYS_EEPROM_I2C_WRITE_1B_DELAY=0
+CONFIG_SYS_EEPROM_I2C_WRITE_2B_DELAY=0
+CONFIG_SYS_EEPROM_I2C_WRITE_3B_DELAY=0
```

Comment:
> Is the check for 0 necessary? The miscutils/Config.src file (down below) defines the default already.
> 
> The only way it could be zero is if the user explicitly set it to zero.

Although the default values have been defined in `miscutils/Config.src`, the `busybox.config` generated by busybox-1.25.1 automatically still not has these values.  This deficiency did not appear in busybox-1.20.2.

The same symptom occurred recently is on `CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS`:

https://github.com/opencomputeproject/onie/blob/master/patches/busybox/feature-onie-syseeprom-command.patch#L863-L870

I did not notice the issue before.  It added the option to all `CONFIG_SYS_EEPROM_DEVICE_I2C=y` platforms (see commits 977abaf and 0cea415) to fix compilation errors.

In the patchset, to prevent from these values resetting by re-generating `busybox.config` in the future, I would add check for 0 to the source rather than modify values in `busybox.config` manually.  Furthermore, setting them to zero explicitly should not make sense to access i2c EEPROM (the original value, 10ms, in `24cXX.c` should be the minimum delay).

What do you think?  Please let me know.  Thank you very much.